test/python/WMComponent_t/WorkflowUpdater_t/WorkflowUpdaterPoller_t.py

```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env python
+
+import os
+import json
+import shutil
+import logging
+import tarfile
+import tempfile
+import unittest
+
+from WMComponent.WorkflowUpdater.WorkflowUpdaterPoller import \
+    blockLocations, checkChanges, updateBlockInfo, writePileupJson, \
+    findJsonSandboxFiles, extractPileupconf, WorkflowUpdaterPoller
+from WMCore.MicroService.Tools.Common import getMSLogger
+from WMCore.WMBase import getTestBase
+from WMCore.Configuration import Configuration
+from WMQuality.Emulators.RucioClient.MockRucioApi import MockRucioApi
```

Comment:
I guess this import is no longer required as well?

Reply:
ok, removed